test-case/check-sof-logger.sh

```diff
@@ -102,18 +102,16 @@ run_loggers()
 
     # This test is not really supposed to run while the DSP is busy at
     # the same time, so $data_file will hopefully not be long.
-    local collect_secs=2
+    local collect_secs=3
+
+    # Delay between logger start and triggering DSP activity
+    local logger_settle_time_secs=1
```

Comment:
I appreciate the long and descriptive name but I don't think this needs to be defined 50 lines before the only place where it's used. Can you keep it all in one place? This is not C89 :-)